libs/wire-api/src/Wire/API/Routes/Internal/Brig/SearchIndex.hs

```diff
@@ -30,3 +31,11 @@ type ISearchIndexAPI =
         :> "refresh"
         :> Post '[JSON] NoContent
     )
+    :<|> Named
+           "update-search-index"
+           ( Summary "force update of search index for a single user"
+               :> "index"
+               :> "update"
+               :> Capture "userId" UserId
+               :> Post '[JSON] NoContent
+           )
```

Comment:
Instead of force updating, we should use the write time of the role to decide the version of the doc.